src/components/NewsGroup.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { DateNewsCard, IconHeading, Button } from '@baltimorecounty/dotgov-components';
+const NewsGroup = (props) => {
+	const { Newsitems = [] } = props;
+	return (
+		<React.Fragment>
+            <div className="row">
+				{Newsitems.map((Newsitem) => (
+                    <div className="col">
```

Comment:
this looks good, but does not respond as expected. At large screens there should be 3 columns like you specified. Between large and medium screens it should be two columns wide. On small and extra small screens it should display in a one column layout. See https://getbootstrap.com/docs/4.3/layout/grid/#grid-options

Let me know if you need help with this.

Reply:
A bottom margin also needs to be added to the cards on smaller screens i can also provide those changes.

Reply:
done